directories/src/redux/reducers/items-reducers.js

```diff
@@ -0,0 +1,37 @@
+import {
+  API_GIVE_PEER_ID,
+  API_START_TRANSACTION,
+  API_SELECT_NEED
+} from '../actions/action-types';
+
+import {
+  API_REQUEST_ERROR
+} from '../../config/';
+
+const initState = {
+  transaction: { txId: '', items: [] },
+  item: {}
```

Comment:
feels confusing having `item` here and `items` inside of transaction. are they related? is it possible to just derive `item` from `items`?